content/docs/sidebar.json

```diff
@@ -100,7 +100,7 @@
         "slug": "how-to",
         "source": false,
         "children": [
-          "add-output-to-stage",
+          "add-dependency-or-output-to-stage",
```

Comment:
Once merged the slug should be add-deps-or-outs-to-a-stage and the doc title should be **Add Dependencies or Outputs to a Stage** but if that's too long for the nav label, let's use something else like "Add Deps/Outs to a Stage".

Reply:
\* Once merged with my changes

Reply:
@jorgeorpinel I merged the branch `how-to`. If you check "file changed" in this PR, it shows 12 files changed. But here (https://github.com/iterative/dvc.org/compare/guide/how-to) it shows 6 files changed (in `guide/how-to` branch) . Is there some problem/conflict? Because the changes in those extra 5-6 files (shown under this PR) have already been merged to master. Please help.

Reply:
Sorry, I don't know what you did in commits 6ecc645 and da629e9. You're gonna have to abandon this branch and just take over PR #1914. Please update the title and description and switch to that branch to apply again all of your changes here @imhardikj.